core/src/main/java/org/elasticsearch/common/io/stream/Writeable.java

```diff
@@ -31,6 +31,11 @@
  *
  * Prefer implementing this interface over implementing {@link Streamable} where possible. Lots of code depends on {@linkplain Streamable}
  * so this isn't always possible.
+ *
+ * The fact that this interface extends {@link StreamableReader} should be consider vestigial. Instead of using its
```

Comment:
I'd love to remove it but that'd make the PR not fit in github's review interface. So I hope to get to more PRs.